src/requirements/Farcaster/FarcasterRequirement.tsx

```diff
@@ -24,6 +25,14 @@ const FarcasterRequirement = (props: RequirementProps) => {
       : undefined
   )
 
```

Comment:
Since we need different hooks for the different types of the req, we should create separate components for these cases, like in PointsRequirement or GuildRequirement. That way we wouldn't need to have all hooks mounted in all cases and the conditions duplicated for them so they don't actually fetch if they don't have to

(We should have done this when we've added channels actually, but it's even more relevant now that we're adding casts too)